Octokit.Tests/Helpers/Arg.cs

```diff
@@ -53,6 +53,21 @@ public static NewRepository NewRepository
             get { return Arg.Is<Dictionary<string, string>>(d => d.Count == 0); }
         }
 
+        public static Dictionary<string, string> DictionaryWithSince
```

Comment:
@SamTheDev if you feel like they are being used many times and the readability is necessary, define them in the classes. Otherwise just inline them.